Makefile

```diff
@@ -125,6 +125,22 @@ fmt-check: required-gofmt-version
 test: fmt-check
 	go test $(PACKAGES)
 
+.PHONY: less-check
+less-check: less
+	@diff=$$(git diff); \
+	if [ -n "$$diff" ]; then \
+		echo "Please run 'make less' and commit the result:"; \
+		echo "$${diff}"; \
+		exit 1; \
+	fi;
+
+.PHONY: less
+less:
```

Comment:
Maybe use `stylesheets-check` in place of `less-check` to be consistent but that would be a personal point and not mandatory. ^^